app/views/devise/sessions/new.html.erb

```diff
@@ -16,8 +16,8 @@
               <%= f.check_box :remember_me, custom: :switch, control_class: "text-secondary"%>
             </div>
           <% end %>
-          <%= f.submit "Log in" %>
-          <%= link_to 'Back', root_path, class: "btn btn-primary" %>
+          <%= f.submit I18n.t("devise.sessions.new.login") %>
```

Comment:
When this is in the view like this, and the path is `devise/sessions/new`, I think you can use a shorthand and just call this `.login`.